.github/workflows/nodejs.yml

```diff
@@ -11,19 +11,20 @@ name: CI
 jobs:
   test:
     name: Test
+    timeout-minutes: 10
```

Comment:
Looks like these tests don't cleanup well. To prevent them running 360 minutes I'm adding this. I'll take a brief look and see if its obvious what's holding them up, but likely that'll be saved for another time.